pkg/config/prow-config-documented.yaml

```diff
@@ -1355,6 +1355,13 @@ scheduler:
         # configured to in the first place.
         mappings:
             "": ""
+    rest_by_job:
+        # Interval is the interval to clean up the cache
+        interval: 0s
```

Comment:
Perhaps we can use a better name for this. Maybe `cache_cleanup_interval` ?

Reply:
will do